functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/JobTabsSpec.groovy

```diff
@@ -0,0 +1,109 @@
+package org.rundeck.tests.functional.selenium.jobs
+
+import org.rundeck.util.annotations.SeleniumCoreTest
+import org.rundeck.util.container.SeleniumBase
+import org.rundeck.util.gui.pages.execution.ExecutionShowPage
+import org.rundeck.util.gui.pages.home.HomePage
+import org.rundeck.util.gui.pages.jobs.JobCreatePage
+import org.rundeck.util.gui.pages.jobs.JobShowPage
+import org.rundeck.util.gui.pages.jobs.JobTab
+import org.rundeck.util.gui.pages.login.LoginPage
+import org.rundeck.util.gui.pages.project.DashboardPage
+import org.rundeck.util.gui.pages.project.ProjectCreatePage
+
+
+@SeleniumCoreTest
+class JobTabsSpec extends SeleniumBase {
+
+    void setup() {
+        def loginPage = go(LoginPage)
+        loginPage.login(TEST_USER, TEST_PASS)
+    }
+
+    void "job nodes tab"() {
+        when:
+            String projectName = "job_nodes_tab"
+            def homePage = page(HomePage)
+            def projectCreatePage = page(ProjectCreatePage)
+            def jobShowPage = page(JobShowPage)
+            def executionShowPage = page(ExecutionShowPage)
+        then:
+            homePage.createProjectButton()
+            projectCreatePage.createProject(projectName)
```

Comment:
you can use the method "setupProject" to use the API instead of UI to create the project, this saves some time and since the test is not intended to test the project creation there is no need to use the UI for that

Reply:
✅ Done